conf/modules.config

```diff
@@ -67,318 +64,297 @@ process {
+        ]]
+    }
 
-        }
-
-        if("wisecondorx" in callers) {
-            withName: "^.*BAM_VARIANT_CALLING_WISECONDORX:WISECONDORX_PREDICT\$" {
-                ext.args = "--seed 0 --bed --plot"
-                memory = { check_max(50.GB * task.attempt, 'memory') }
-                ext.prefix = { "${meta.id}.wisecondorx" }
-                publishDir = [[
-                    enabled: !output_callers,
-                    overwrite: true,
-                    path: { "${params.outdir}/${meta.id}" },
-                    mode: params.publish_dir_mode,
-                    saveAs: { filename -> filename != "versions.yml" ? filename : null }
-                ],[
-                    enabled: output_callers,
-                    overwrite: true,
-                    path: { "${params.outdir}/${meta.id}/wisecondorx" },
-                    mode: params.publish_dir_mode,
-                    saveAs: { filename -> filename != "versions.yml" ? filename : null }
-                ]]
-            }
-        }
-
-        withName: "^.*BAM_CNV_CALLING:[^:]*:BEDGOVCF\$" {
-            ext.prefix = { "${meta.id}.${meta.caller}" }
-            ext.args = { meta.caller == "qdnaseq" ? "--skip 1" : "--header" }
-            publishDir = [[
-                enabled: !params.annotate && foundCnvCallers.size() == 1 && (count_types == 1 || !params.concat_output),
-                overwrite: true,
-                path: { "${params.outdir}/${meta.id}" },
-                mode: params.publish_dir_mode,
-                saveAs: { filename -> filename.endsWith("vcf.gz") ? filename.replace("${meta.caller}", "cnv") : null }
-            ],[
-                enabled: output_callers,
-                overwrite: true,
-                path: { "${params.outdir}/${meta.id}/${meta.caller}" },
-                mode: params.publish_dir_mode,
-                saveAs: { filename -> filename != "versions.yml" ? filename : null }
-            ]]
-        }
-
-        withName: "^.*BAM_CNV_CALLING:[^:]*:TABIX_TABIX\$" {
-            publishDir = [[
-                enabled: !params.annotate && foundCnvCallers.size() == 1 && (count_types == 1 || !params.concat_output),
-                overwrite: true,
-                path: { "${params.outdir}/${meta.id}" },
-                mode: params.publish_dir_mode,
-                saveAs: { filename -> filename.endsWith("vcf.gz.tbi") ? filename.replace("${meta.caller}", "cnv") : null }
-            ],[
-                enabled: output_callers,
-                overwrite: true,
-                path: { "${params.outdir}/${meta.id}/${meta.caller}" },
-                mode: params.publish_dir_mode,
-                saveAs: { filename -> filename.endsWith(".tbi") ? filename : null }
-            ]]
-        }
+    withName: "^.*BAM_VARIANT_CALLING_WISECONDORX:WISECONDORX_PREDICT\$" {
+        ext.args = "--seed 0 --bed --plot"
+        memory = { 50.GB * task.attempt }
```

Comment:
50GB??

Reply:
We're talking about WGS data here ;). But yeah it's a bit high to be on the safe side for now